aspnetcore/data/ef-rp/intro.md

```diff
@@ -257,6 +257,8 @@ In the *Data* folder, create a new class file named *DbInitializer.cs* and add t
 
 [!code-csharp[](intro/samples/cu21/Data/DbInitializer.cs?name=snippet_Intro)]
 
+Note: The preceding code uses `Models` for the namespace (`namespace ContosoUniversity.Models`) rather than `Data`. `Models` is used to be consistent with the scaffolder generated code. For more information, see [this Github scaffolding issue](https://github.com/aspnet/Scaffolding/issues/822).
```

Comment:
I think the adj would be "scaffolder-generated" ... but it's a nittola. It can go without the state verb, too, if you like ...

```suggestion
Note: The preceding code uses `Models` for the namespace (`namespace ContosoUniversity.Models`) rather than `Data`. `Models` is consistent with the scaffolder-generated code. For more information, see [this Github scaffolding issue](https://github.com/aspnet/Scaffolding/issues/822).
```